content/docs/guides/liquibase.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Get started with Liquibase and Neon
+subtitle: Learn how to manage schema changes in Neon with Liquibase
+enableTableOfContents: true
+---
+
+Liquibase is an open-source library for tracking, managing, and applying database schema changes. To learn more about Liquibase, refer to the [Liquibase documentation](https://docs.liquibase.com/home.html).
+
+This guide steps you through installing the Liquibase CLI, configuring Liquibase to connect to a Neon database, deploying a database schema change, and rolling back the schema change. The guide follows the setup described in the [Liquibase Get Started](https://www.liquibase.org/get-started/quickstart).
+
+## Prerequisites
+
+- A Neon account. See [Sign up](/docs/get-started-with-neon/signing-up).
+- A Neon project. See [Create your first project](/docs/get-started-with-neon/setting-up-a-project).
+- Liquibase requires Java. To check if you have Java installed, run `java --version`. If not, refer to the installation instructions for your operating system.
```

Comment:
```suggestion
- Liquibase requires Java. To check if you have Java installed, run `java -version`. If not, refer to the installation instructions for your operating system.
```

Reply:
On my machine, I got an error with two dashes.

`barrygrenon@barrys-mbp ~ % java -version
java version "1.8.0_391"
Java(TM) SE Runtime Environment (build 1.8.0_391-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.391-b13, mixed mode)
barrygrenon@barrys-mbp ~ % java --version
Unrecognized option: --version
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.`